src/NSwag.Commands/Commands/SwaggerGeneration/TypesToJsonSchemaCommand.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System.Collections.Generic;
```

Comment:
Move this to the /Generation directory

Reply:
will tackle in single pass as soon as https://github.com/RicoSuter/NJsonSchema/pull/981 and may be other pull (for flat ref) considered. thanks